index.html

```diff
@@ -11,20 +11,31 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
+<body>
+
```

Comment:
- [optional] Great job implementing the previous reviewer's changes. kindly make the mobile popup menu not function when you switch to the desktop version. 
![Screenshot (174)](https://user-images.githubusercontent.com/65298048/175104680-ceb3267c-d910-482b-bc76-5e75b25bc79b.png)